ORT Scanner: invalid spec

Adding scanner into the run parameter results in 
```
 Running ORT Scanner... 
docker: invalid spec: :/ssh.socket: empty section between colons.
See 'docker run --help'.
Error: Process completed with exit code 125.
```
Used configuration as
```yaml
name: Ort
on: 
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]
permissions:
  contents: read
jobs:
  test:
    runs-on: ubuntu-latest
    timeout-minutes: 5
    steps:
      - uses: actions/checkout@v4
      - run: git config --global url.https://github.com/.insteadOf ssh://git@github.com/
      - uses: oss-review-toolkit/ort-ci-github-action@v1
        with:
          run: 'cache-dependencies,analyzer,scanner,evaluator,advisor,reporter,upload-results'
      
```

--- a/action.yml
+++ b/action.yml
@@ -426,6 +426,7 @@ runs:
       if: contains(inputs.run, 'scanner')
       run: |
         echo -e "\e[1;33m Running ORT Scanner... "
+        eval "$(ssh-agent -s)" \
         docker run \
         --mount type=bind,source=$HOME,target=/home/ort \
         --mount type=tmpfs,target=/tmp \

